bump_packaging

```diff
@@ -51,7 +51,9 @@ fi
 git commit -m "Release $FULLVERSION"
 
 if [[ $PACKAGING_PR == true ]] ; then
-	git phr -m "Release $FULLVERSION" -b "$REMOTE_BRANCH"
+	# TODO: assumes a remote named whoami for the user's fork
+	git push --set-upstream "$(whoami)" HEAD
+	hub pull-request -m "Release $FULLVERSION" -b "$REMOTE_BRANCH"
```

Comment:
I've been using [`gh`](https://github.com/cli/cli) and there it's `gh pr create` and it will take care of pushing and everything.

Reply:
Somehow I thought `gh` was written in Javascript and always ignored it, but now I see it's not and it's even packaged in Fedora. Maybe I'll take some time to look at converting.

Reply:
Yeah, I'm using the Fedora package and it's quite OK for the things I needed it for.

Reply:
I noticed you often push to the original repository instead of your own fork, which is something I'd like to avoid. Is that because of `gh` behavior?

Reply:
No, that's my lazyness, you can tell it where to push to. I just often don't even have an own fork (configured)